specification/iothub/resource-manager/Microsoft.Devices/preview/2019-03-22-preview/iothub.json

```diff
@@ -2534,6 +2541,33 @@
         "isEnabled"
       ]
     },
+    "EnrichmentProperties": {
+      "description": "The properties of an enrichment that your IoT hub applies to messages to endpoints.",
+      "type": "object",
+      "properties": {
+        "key": {
+          "description": "The key for the enrichment property.",
```

Comment:
Should we explain more? Like this is the key in the app property? Or re-direct to a documentation?

Reply:
@ash2017 ?

Reply:
we can mention key or name of the enrichment property.